activefires_pp/tests/test_messaging.py

```diff
@@ -40,7 +40,7 @@
 TEST_MSG_TXT = """pytroll://VIIRS/L2/AFI/edr/2/nrk/test/polar/direct_readout file safusr.t@lxserv2313.smhi.se 2023-07-05T10:27:28.821803 v1.01 application/json {"start_time": "2023-07-05T10:07:50", "end_time": "2023-07-05T10:09:15", "orbit_number": 1, "platform_name": "Suomi-NPP", "sensor": "viirs", "format": "edr", "type": "txt", "data_processing_level": "2", "variant": "DR", "orig_orbit_number": 60553, "origin": "172.29.4.164:9099", "uri": "/san1/polar_out/direct_readout/viirs_active_fires/unfiltered/AFIMG_npp_d20230705_t1007509_e1009151_b60553_c20230705102721942345_cspp_dev.txt", "uid": "AFIMG_npp_d20230705_t1007509_e1009151_b60553_c20230705102721942345_cspp_dev.txt"}"""  # noqa
 
 
-def get_fake_publiser(portnumber=1979):
+def get_fake_publisher(portnumber=1979):
```

Comment:
These changes are also present in the other pull request...

Reply:
But I needed them here for the CI tests, they started failing after posttroll was updated in the CI. They did not fail before with the my old code and old posttroll

Reply:
ok, #21 is merged now, so it can be merged into this branch also